ini_multiline_value_fix_test.go

```diff
@@ -0,0 +1,60 @@
+package ini_test
+
+import (
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+	"path/filepath"
+	"testing"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+}
+
+func TestMultiline(t *testing.T) {
+	Value1 := "some text here\n"+
+		"some more text here\n"+
+		"\n"+
+		"there is an empty line above and below\n"
+
+	Value2 := "there is an empty line above\n"+
+		"that is not indented so it should not be part\n"+
+		"of the value"
```

Comment:
These two values only be used once, at equality check, I think directly put string literals in `So(...` is fine.

Reply:
I will adopt this change as requested.